x/gov/tags/tags.go

```diff
@@ -1,14 +1,17 @@
-// nolint
 package tags
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+// Governance tags
 var (
 	ActionProposalDropped  = []byte("proposal-dropped")
 	ActionProposalPassed   = []byte("proposal-passed")
 	ActionProposalRejected = []byte("proposal-rejected")
+	ActionProposalCreated  = []byte("proposal-created")
```

Comment:
(I think) we’re now using the message type as the action for this 3 tags ? Need to double check

Reply:
Not sure I follow? Before we weren't using any action at all!